run-dljc.sh

```diff
@@ -6,14 +6,16 @@
 ### Usage
 
 # - Move this script to an experiment directory.
-# - Make a file containing a list of git repositories, one per line. Repositories must be of the form: https://github.com/username/repository - the script is reliant on the number of slashes, so excluding https:// is an error.
+# - Make a file containing a list of git repositories, one per line.
```

Comment:
This contradicts later documentation in this file for the `-i` flag that says that the file should also contain git commit hashes, and possibly a third column.